### ISSUE TITLE
"Open With Pulsar" context option doesn't go away after uninstalling

### ISSUE THREAD

**Issue by Robert-W-Ward** [pulsar-edit/pulsar#274]
### Thanks in advance for your bug report!

- [X] Have you reproduced issue in [safe mode](https://pulsar-edit.dev/docs/atom-archive/faq/sections/what-does-safe-mode-do.html)?
- [X] Have you used the [debugging guide](https://pulsar-edit.dev/docs/atom-archive/hacking-atom/#debugging) to try to resolve the issue?
- [X] Have you checked our [FAQs](https://pulsar-edit.dev/docs/launch-manual/sections/faq/) to make sure your question isn't answered there?
- [X] Does your issue already [exist](https://github.com/issues?q=is%3Aissue+user%3Apulsar-edit+is%3Aopen)?
- [X] Have you checked you are on the latest release of Pulsar?

### What happened?

After uninstalling Pulsar the "Open With pulsar" context menu is still present on right click.

### Which OS does this happen on?

🪟 Windows

### OS details

Windows 10 Home 21H2 19044.2364

### Which CPU architecture are you running this on?

64-bit(x86_64)

### What steps are needed to reproduce this?

Install 1.100.0-beta version
go into settings and turn on "Show in file context menus"
turning it off doesn't change anything 
uninstall and have broken context menu option.

**Comment by confused-Techie** [comment]
Thanks for reporting this bug. I'll try to look into it, but seems strange this is broken as I don't think we have changed that section of code other than rebranding

**Comment by icecream17** [comment]
I think this also happens with the original Atom tbh...

**Comment by bobemoe** [comment]
Yeah looks like known issue of Atom not removing registry entries: https://github.com/atom/atom/issues/21309

**Comment by confused-Techie** [comment]
> Yeah looks like known issue of Atom not removing registry entries: [atom/atom#21309](https://github.com/atom/atom/issues/21309)

Appreciate you doing the research on this, so we at least know this isn't a regression on our part.

But something we still will likely want to resolve, appreciate the contribution!

**Comment by savetheclocktower** [comment]
@confused-Techie, do you think this is still an issue? I know you've given attention to the post-install script since this issue was opened.

**Comment by confused-Techie** [comment]
@savetheclocktower Thanks for the ping, this one is not currently resolved.

The only thing we remove during uninstall on Windows, is removing Pulsar from the systems PATH.

We should probably consider #1403 for this one
